helium/src/main/groovy/com/stanfy/helium/handler/codegen/java/entity/AndroidParcelableWriter.java

```diff
@@ -109,6 +113,13 @@ public void writeClassEnd(final Message message) throws IOException {
     output.emitEmptyLine();
     output.emitAnnotation(Override.class);
     output.beginMethod("void", "writeToParcel", EnumSet.of(Modifier.PUBLIC), ANDROID_OS_PARCEL, "dest", "int", "options");
+
+    // we assume that if parent is specified, it's also a parcelable
+    // note: may be insert safe type check Parcelable.class.isAssignableFrom(getClass().getSupperClass())
```

Comment:
You can have no information about classes at code generation step. That class can be not compiled yet or simply not present in our classpath.